src/Maestro/Maestro.Data/Migrations/BuildAssetRegistryContextModelSnapshot.cs

```diff
@@ -572,6 +602,19 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .OnDelete(DeleteBehavior.Cascade);
                 });
 
+            modelBuilder.Entity("Maestro.Data.Models.ChannelReleasePipeline", b =>
+                {
+                    b.HasOne("Maestro.Data.Models.Channel", "Channel")
+                        .WithMany("ChannelReleasePipelines")
+                        .HasForeignKey("ChannelId")
+                        .OnDelete(DeleteBehavior.Cascade);
```

Comment:
Cascade is a very dangerous option. :-) Do we really want to do that? It can tend to delete way more things you think without meaning to, which is usually a bad idea in a production database with no way to recover the accidentally deleted information.  I'd rather the delete fail that implicitly delete other things

Reply:
done